### PR TITLE
Crashlytics Release Notes from Git Commits

### PR DESCRIPTION
**Pull request by eisforellen**
This PR adds git commits to the release notes in Crashlytics by defaulting to using changelog_from_git_commits.

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -55,21 +55,23 @@ platform :ios do
     crashlytics(
       crashlytics_path: './Pods/Crashlytics/',
       groups: options[:group],
-      notifications: 'true'
+      notifications: 'true',
+      notes: options[:notes]
```
Comment (jshier) [inline fastlane/Fastfile]: You need to handle the case where the user doesn't want notes. Perhaps a Rubyist would be able to come up with an elegant way to handle passing the parameter or not.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -55,21 +55,23 @@ platform :ios do
...
+    options[:notes] ||= changelog_from_git_commits
+
     raise "beta: A Crashlytics group must be passed as an option." unless options[:group]
-    
+
     build(configuration: options[:configuration], include_bitcode: options[:include_bitcode], export_method: options[:export_method])
-    
-    uploadToCrashlytics(group: options[:group])
+
+    uploadToCrashlytics(group: options[:group], notes: options[:notes])
```
Comment (jshier) [inline fastlane/Fastfile]: Again, if the user doesn't pass any notes, we don't want to call this with a notes parameter.

File: fastlane/Fastfile (hunk 1 above)
Comment (eisforellen) [inline fastlane/Fastfile]: @jshier What about having them default to an empty string? Or would you prefer we not include the notes parameter if there are no notes being passed in?

Comment (jshier) [inline fastlane/Fastfile]: Pretty sure we want to not have the parameter at all but more investigation is required.

**Comment by eisforellen** [comment]
@jshier I changed it so that changelog_from_git_commits is only called if a user passes in that parameter. According to the [docs](https://docs.fastlane.tools/getting-started/ios/beta-deployment/#release-notes) Crashlytics will [automatically include the changelog](https://docs.fastlane.tools/getting-started/android/beta-deployment/#release-notes). Let me know your thoughts.

**Pull request merged**